azure-mgmt-storage/src/test/java/com/microsoft/azure/management/storage/StorageAccountNetworkRuleTests.java

```diff
@@ -33,59 +33,59 @@ public void canConfigureNetworkRulesWithCreate() throws Exception {
         String SA_NAME3 = generateRandomResourceName("javacsmsa", 15);
         String SA_NAME4 = generateRandomResourceName("javacsmsa", 15);
 
-        StorageAccount storageAccount1 = storageManager.storageAccounts()
-                .define(SA_NAME1)
-                .withRegion(Region.US_EAST)
-                .withNewResourceGroup(RG_NAME)
-                .create();
-
-        Assert.assertNotNull(storageAccount1.networkSubnetsWithAccess());
-        Assert.assertEquals(0, storageAccount1.networkSubnetsWithAccess().size());
-
-        Assert.assertNotNull(storageAccount1.ipAddressesWithAccess());
-        Assert.assertEquals(0, storageAccount1.ipAddressesWithAccess().size());
-
-        Assert.assertNotNull(storageAccount1.ipAddressRangesWithAccess());
-        Assert.assertEquals(0, storageAccount1.ipAddressRangesWithAccess().size());
-
-        Assert.assertTrue(storageAccount1.isAccessAllowedFromAllNetworks());
-        Assert.assertTrue(storageAccount1.canAccessFromAzureServices());
-        Assert.assertTrue(storageAccount1.canReadMetricsFromAnyNetwork());
-        Assert.assertTrue(storageAccount1.canReadMetricsFromAnyNetwork());
-
-        ResourceGroup resourceGroup = resourceManager
-                .resourceGroups()
-                .getByName(storageAccount1.resourceGroupName());
-
-        StorageAccount storageAccount2 = storageManager.storageAccounts()
-                .define(SA_NAME2)
-                .withRegion(Region.US_EAST)
-                .withExistingResourceGroup(resourceGroup)
-                .withAccessFromIpAddress("23.20.0.0")
-                .create();
-
-        Assert.assertNotNull(storageAccount2.inner().networkRuleSet());
-        Assert.assertNotNull(storageAccount2.inner().networkRuleSet().defaultAction());
-        Assert.assertNotNull(storageAccount2.inner().networkRuleSet().defaultAction().equals(DefaultAction.DENY));
-
-        Assert.assertNotNull(storageAccount2.networkSubnetsWithAccess());
-        Assert.assertEquals(0, storageAccount2.networkSubnetsWithAccess().size());
-
-        Assert.assertNotNull(storageAccount2.ipAddressesWithAccess());
-        Assert.assertEquals(1, storageAccount2.ipAddressesWithAccess().size());
-
-        Assert.assertNotNull(storageAccount2.ipAddressRangesWithAccess());
-        Assert.assertEquals(0, storageAccount2.ipAddressRangesWithAccess().size());
-
-        Assert.assertFalse(storageAccount2.isAccessAllowedFromAllNetworks());
-        Assert.assertFalse(storageAccount2.canAccessFromAzureServices());
-        Assert.assertFalse(storageAccount2.canReadMetricsFromAnyNetwork());
-        Assert.assertFalse(storageAccount2.canReadMetricsFromAnyNetwork());
+//        StorageAccount storageAccount1 = storageManager.storageAccounts()
```

Comment:
why is all of this commented out? 
If the code is not relevant anymore, kindly remove it.